extensions/BMO/template/en/default/bug/create/create-ipc.html.tmpl

```diff
@@ -0,0 +1,267 @@
+          <strong>Does your snippet need to be targeted to a particular country? If so, please specify.</strong>
+        </label>
+      </div>
+      <input type="text" name="target_country" id="target_country" size="80">
+    </div>
+
+    <p>
+    If you would like your snippet to include a link, please tag it by using the URL builder <a href="http://bit.ly/1JOgDr1">(http://bit.ly/1JOgDr1)</a> and following the steps below:
+    </p>
+    <img src="https://bug1298932.bmoattachments.org/attachment.cgi?id=8786064" style = "width:50%"><br><br>
+    <div class="row">
+      <div class="row_desc">
+        <label for="tagged_link">
+          <strong>Insert your tagged link here.</strong>
+        </label>
+      </div>
+      <input type="text" name="tagged_link" id="tagged_link" size="80">
+    </div>
+
+    <p>
+    Please reference the Mozilla In-Product Messaging Calendar for available dates. <a href="http://bit.ly/2aZ3w9c"> http://bit.ly/2aZ3w9c</a>
+    </p>
+
+    <div class="row">
+      <div class="row_desc">
+        <label class="required" for="start_run">
+          <strong>When would you like your snippet to start its run?</strong>
+        </label>
+      </div>
+      <input required class="date_field" name="start_run" id="start_run">
+      <img class="date_field-img" id="start_run-img" src="extensions/BugModal/web/calendar.png" width="16" height="16">
+    </div>
+
+    <div class="row">
+      <div class="row_desc">
+        <label class="required" for="complete_run">
+          <strong>When would you like your snippet to complete its run?</strong>
+        </label>
+      </div>
+      <input required class="date_field" name="complete_run" id="complete_run">
+      <img class="date_field-img" id="complete_run-img" src="extensions/BugModal/web/calendar.png" width="16" height="16">
+    </div>
+
+    <div class="row">
+      <div class="row_desc">
+        <strong>Will you be able to provide the following assets for your snippet? Check all that apply.</strong><br> Feel free to look at our existing icon library:  <a href="http://bit.ly/2aWTmqT">http://bit.ly/2aWTmqT</a>
+      </div>
+      <input type="radio" name="snippet_assets" id="copy" value="Copy">
+      <label for="copy">Copy</label><br>
+      <input type="radio" name="snippet_assets" id="icon" value="Icon">
+      <label for="icon">Icon</label><br>
+      <input type="radio" name="snippet_assets" id="special_format" value="Special Format">
+      <label for="special_format">Special Format</label><br>
+      <input type="radio" name="snippet_assets">
+      <label for="others">Others: </label>
+      <input type="text" name="snippet_assets" id="others" size="30">
+    </div>
+
+    <h3><strong>Contact Info</strong></h3>
+
+    <div class="row">
+      <div class="row_desc">
+        <label class="required" for="name">
+          <strong>Your Name</strong>
+        </label>
+      </div>
+      <input type="text" required name="name" id="name" size="80">
```

Comment:
add value="[% user.name FILTER html %]" to default to the person's bugzilla name.